nsinit/exec.go

```diff
@@ -51,7 +45,7 @@ func execAction(context *cli.Context) {
 	}
 	process := &libcontainer.Process{
 		Args:   context.Args(),
-		Env:    context.StringSlice("env"),
+		Env:    append(os.Environ(), context.StringSlice("env")...),
```

Comment:
I agree that this is little unexpected behavior by default. `--inherit-env` flag makes sense.